crates/ruff_db/src/diagnostic/mod.rs

```diff
@@ -649,6 +649,20 @@ impl DiagnosticId {
         })
     }
 
+    /// Returns a "concise" description of this diagnostic ID.
```

Comment:
How involved is it to change `as_str`? Is `as_str` even still used in other places?

Reply:
Nice catch. Removing this revealed that there was still a `lint:` in the diagnostic output.